tests/integration/azure_blob/test_azure_blob.py

```diff
@@ -76,32 +77,74 @@
 class TestAzureBlob:
     @classmethod
     def setup_class(cls):
+        cls.assets_path = join(dirname(dirname(abspath(__file__))), "assets")
         cls.profile_name = "azure_blob_ds_profile"
         cls.test_dir = "test_mlrun_azure_blob"
+        cls.run_dir = cls.test_dir + f"/run_{uuid.uuid4()}"
         cls._bucket_name = config["env"].get("AZURE_CONTAINER", None)
-        cls.test_file = here / "test.txt"
+        cls.test_file = join(cls.assets_path, "test.txt")
         with open(cls.test_file) as f:
             cls.test_string = f.read()
+        cls._azure_fs = None
 
-    def teardown_method(self):
-        test_dir = f"{self._bucket_name}/{self.test_dir}"
-        if not self._azure_fs:
+    @classmethod
+    def teardown_class(cls):
+        test_dir = f"{cls._bucket_name}/{cls.test_dir}"
+        if not cls._azure_fs:
             return
-        if self._azure_fs.exists(test_dir):
-            self._azure_fs.delete(test_dir, recursive=True)
+        if cls._azure_fs.exists(test_dir):
+            cls._azure_fs.delete(test_dir, recursive=True)
             logger.debug("test directory has been cleaned.")
 
+    @classmethod
+    def create_fs(cls, storage_options):
+        #  Create filesystem object only once - for teardown_class method.
+        if not cls._azure_fs:
+            azure_fs = AzureBlobFileSystem(storage_options)
+            try:
+                azure_fs.info(cls._bucket_name)  # in order to check connection ...
+                cls._azure_fs = azure_fs
+            except Exception:
+                pass
```

Comment:
How come we just ignore any exceptions?

Reply:
added warning log, also in test_v3io.py

Reply:
No one will read this. Why not fail the test?